modular_skyrat/master_files/code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -11,7 +11,7 @@
 	return ..()
 
 /datum/status_effect/crusher_mark/on_apply()
-	if(owner.mob_size >= MOB_SIZE_LARGE)
+	if(!HAS_TRAIT(owner, TRAIT_CRUSHER_MARK_IMMUNE) && owner.mob_size >= MOB_SIZE_LARGE)
```

Comment:
More efficient to check it second since comparatively less mobs will pass the size check

```suggestion
	if(owner.mob_size >= MOB_SIZE_LARGE && !HAS_TRAIT(owner, TRAIT_CRUSHER_MARK_IMMUNE))
```

Reply:
Fair point